File: wrappers/java/src/main/java/org/hyperledger/indy/sdk/InvalidStructureException.java
package org.hyperledger.indy.sdk;

/**
 * Exception thrown when a value passed to the SDK was not structured so that the SDK could correctly process it.
 */
public class InvalidStructureException extends IndyException
{
	private static final long serialVersionUID = -2157029980107821313L;
	private final static String message = "A value being processed is not valid.";

	/**
	 * Initializes a new InvalidStructureException.
	 */
	public InvalidStructureException()
	{
		super(message, ErrorCode.CommonInvalidStructure.value());
	}
}

File: wrappers/dotnet/indy-sdk-dotnet/InvalidStructureException.cs
namespace Hyperledger.Indy
{
    /// <summary>
    /// Exception indicating that a value being processed was not considered a valid value.
    /// </summary>
    public class InvalidStructureException : IndyException
    {
        const string message = "A value being processed is not valid.";

        /// <summary>
        /// Initializes a new InvalidStructureException.
        /// </summary>
        internal InvalidStructureException() : base(message, (int)ErrorCode.CommonInvalidStructure)
        {

        }
    }

}
